engine/runtime-instrument-common/src/main/scala/org/enso/interpreter/instrument/job/AnalyzeModuleInScopeJob.scala

```diff
@@ -41,6 +41,7 @@ final class AnalyzeModuleInScopeJob(
     ctx: RuntimeContext
   ): Unit = {
     if (!module.isIndexed && module.getSource != null) {
+      module.indexing()
```

Comment:
Here we are setting an internal `module` field and leaving `synchronized` section...